database/seeds/RolesAndPermissionsSeeder.php

```diff
@@ -22,37 +21,70 @@ public function run()
         $adminPermissions[] = Permission::firstOrCreate([ 'name' => 'rooms.delete' ])->id;
 
         $adminPermissions[] = Permission::firstOrCreate([ 'name' => 'settings.manage' ])->id;
-        $adminPermissions[] = Permission::firstOrCreate([ 'name' => 'settings.viewAny' ])->id;
-        $adminPermissions[] = Permission::firstOrCreate([ 'name' => 'settings.update' ])->id;
+
+        $adminPermissions[] = Permission::firstOrCreate([ 'name' => 'applicationSettings.viewAny' ])->id;
```

Comment:
> Renaming in seeds is not the best idea since the previously created permissions will stay in the database!

See line 58+59, it addresses this issue